src/video_core/texture_cache/util.cpp

```diff
@@ -257,7 +258,7 @@ template <u32 GOB_EXTENT>
 }
 
 [[nodiscard]] constexpr LevelInfo MakeLevelInfo(PixelFormat format, Extent3D size, Extent3D block,
-                                                u32 tile_width_spacing) {
+                                                u32 tile_width_spacing, const ImageInfo& info) {
```

Comment:
```suggestion
                                                u32 tile_width_spacing, s32 num_levels) {
```

Since this is how other parameters are already handled.

Reply:
With this nightmare code, that can't work as then info becomes undefined.

Reply:
Yes, this is indeed not the only line you need to change, but the required modification is borderline trivial.

Reply:
think i found a way